docs/en/week12/12-1.md

```diff
@@ -0,0 +1,221 @@
+---
+lecturer: Marc'Aurelio Ranzato
+title: Low Resource Machine Translation (Part 1)
+authors: Fanzeng Xia, Rohith Mukku
+date: 21 April 2021
+---
+
+# Low Resource Machine Translation
+
+## Neural Machine Translation (NMT)
+Neural Machine Translation (NMT) is an end-to-end learning approach for automated translation, with the potential to overcome many of the weaknesses of conventional phrase-based translation systems. Its architecture typically consists of two parts, one to consume the input text sequence (encoder) and one to generate translated output text (decoder). NMT is often accompanied by an attention mechanism which helps it cope effectively with long input sequences. The decoder learns to (soft) align via attention. A translation example from Italian to English is shown in the figure below:
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-1/figure1.png" style="background-color:#DCDCDC;"><br>
+<b>Figure 1:</b> A translation example from Italian to English
+</center>
+
+### Parallel Dataset
+A parallel dataset contains a collection of original texts in language L1 and their translations into languages L2 (or consist of texts of more than two languages). As shown in the figure below, it can be used as labelled data to train NMT systems.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-1/figure2.png" style="background-color:#DCDCDC;"><br>
+<b>Figure 2:</b> A parallel dataset consist of Italian and English
+</center>
+ 
+### Train NMT
+The standard way of training NMT is by using maximum likelihood. Give a source sentence $x$ and the model parameters $\theta$, we will seek to maximize the log likelihood of the joint probability of all the ordered sequence of tokens in the target sentence $y$, as illustrated by the equation below:
+
+$$ - \log\ p(y|x; \theta) = - \sum_{j=1}^{n} log\ p(y_j|y_{j-1},...y_1,x; \theta) $$
```

Comment:
`log` needs `\` before.
The conditional bar is `\mid`.
Vectors need `\vect{}`.

Here and elsewhere.